lib/mayaUsd/render/vp2RenderDelegate/material.cpp

```diff
@@ -497,6 +385,9 @@ _LoadUdimTexture(const std::string& path, bool& isColorSpaceSRGB, MFloatArray& u
 MHWRender::MTexture*
 _LoadTexture(const std::string& path, bool& isColorSpaceSRGB, MFloatArray& uvScaleOffset)
 {
+    MProfilingScope profilingScope(
```

Comment:
Adding some profiler event to help identifying bottlenecks.

In the NV attic scene, the biggest bottleneck is texture loading actually which takes 54 seconds. The optimization of this commit reduces material translation and compilation, improves consolidation, but won't help with texture loading overhead. The first frame drops from the original 70s to 54s, while HdSt takes only 30+ seconds. I think we need to understand why texture loading is slower than HdSt, maybe it is a task for next release.

Reply:
FWIW, NV Attic scene contains 188 4k textures and a dozen of smaller textures.